detox/detox.d.ts

```diff
@@ -1329,17 +1335,17 @@ declare global {
             element(systemMatcher: SystemMatcher): IndexableSystemElement;
         }
 
-        interface DetoxCopilotFacade extends Pick<CopilotFacade, "perform"> {
+        interface DetoxPilotFacade extends Pick<Pilot, "perform", "autopilot", "extendAPICatalog"> {
```

Comment:
```suggestion
        interface DetoxPilotFacade extends Pick<Pilot, "perform" | "autopilot" | "extendAPICatalog"> {
```

Incorrect syntax, see https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Reply:
fixed